src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -36,7 +36,10 @@
   private final String territoryName;
   private final IUIContext uiContext;
   private final static List<String> STATIC_UNITS = Arrays.asList("aaGun","factory");//TODO add every "static" non-movable unit type of every map.
-  private static byte drawUnitNationMode = 2;
+  private static UnitFlag drawUnitNationMode = UnitFlag.NEXT_TO;
```

Comment:
I suggest `UnitFlagDrawMode` for an enum name, more precise.
Also please update the variable name so it matches the enum.